docs/en/rst/api/core/v1/attachment.rst

```diff
@@ -195,6 +195,9 @@ is_private        boolean  ``true`` if the attachment should be private
                            not specified.
 flags             array    Flags objects to add to the attachment. The object
                            format is described in the Flag object below.
+bug_flags         array    Flag objects to add to the attachment's bug. See the
+                           ``flags`` param for :ref:`rest_create_bug` for the
+                           objecct format.
```

Comment:
s/objecct/object/

Reply:
Aha, thanks for fixing! Not sure why my spellchecker didn’t flag it.